tests/helpers/test_entity_registry.py

```diff
@@ -1431,7 +1562,7 @@ async def test_remove_config_entry_from_device_removes_entities_2(
         config_entry_2.entry_id,
     }
 
-    # Create one entity for each config entry
+    # Create an entity without config entry
```

Comment:
Side note: It's weird to have an entity entry connected to a device entry but not connected to the config entry of the device entry. Should that even be allowed?